keyboards/gray_studio/hb85/keymaps/via/rules.mk

```diff
@@ -0,0 +1,4 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+
+MOUSEKEY_ENABLE = no
```

Comment:
Is this required now we have a 3rd endpoint?

Reply:
You are correct!